secboot/secboot_tpm_test.go

```diff
@@ -345,53 +343,37 @@ func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncrypted(c *C) {
 		disk                *disks.MockDiskMapping
 	}{
 		{
-			// happy case with tpm and encrypted device (lock requested)
-			tpmEnabled: true, hasEncdev: true, lockRequest: true, lockOk: true,
+			// happy case with tpm and encrypted device
+			tpmEnabled: true, hasEncdev: true,
 			activated:       true,
 			disk:            mockDiskWithEncDev,
 			expUnlockMethod: secboot.UnlockedWithSealedKey,
 		}, {
-			// happy case with tpm and encrypted device (lock requested) with
-			// an alternative keyfile
-			tpmEnabled: true, hasEncdev: true, lockRequest: true, lockOk: true,
+			// happy case with tpm and encrypted device
+			// with an alternative keyfile
+			tpmEnabled: true, hasEncdev: true,
 			activated:       true,
 			disk:            mockDiskWithEncDev,
 			keyfile:         "some-other-keyfile",
 			expUnlockMethod: secboot.UnlockedWithSealedKey,
 		}, {
-			// device activation fails (lock requested)
-			tpmEnabled: true, hasEncdev: true, lockRequest: true, lockOk: true,
+			// device activation fails
+			tpmEnabled: true, hasEncdev: true,
```

Comment:
nitpick, but this test case is more clear if we explicitly set `activated: false` but that's not endemic to this pr specifically